bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Cookie.java

```diff
@@ -136,11 +137,27 @@ private boolean verifyLedgerDirs(Cookie c, boolean checkIfSuperSet) {
         }
     }
 
+    private boolean verifyDefaultIndexDirsWithLedgerDirs(String indexDirsTpm, Cookie c, boolean checkIfSuperSet) {
```

Comment:
we can change this method name -> doVerifyIndexDirs(String indexDirs, boolean checkIfSuperSet). 
In this method, we pass in the indexDirs from rm cookie or local file cookie. I check the cookie's param indexDirs is from origin indexDirs or ledgersDir.

Reply:
good suggestion,updated